src/common/addlayers/ServerService.js

```diff
@@ -179,6 +203,63 @@ var SERVER_SERVICE_USE_PROXY = true;
       } else {
         serverInfo.isVirtualService = false;
       }
+
+      return serverInfo;
+    };
+
+    this.getWfsRequestUrl = function(url) {
+      var wfsurl = null;
+      var server = service_.getServerByUrl(url);
+      url = goog.isDefAndNotNull(server) ? service_.getMostSpecificUrl(server) : url;
+      var currentDomain = locationService_.host();
+      if (goog.isDefAndNotNull(url)) {
+        if (url.indexOf(currentDomain) > -1) {
+          wfsurl = 'http://' + currentDomain + ':8000/wfsproxy/';
```

Comment:
@davisc Thank you. I fixed this by using `window.location.host`, which returns both the host and port number, unlike the Angular location service.